src/components/Firestore/utils.ts

```diff
@@ -213,3 +215,7 @@ function latStr(lat: number): string {
 function longStr(long: number): string {
   return `${Math.abs(long)}° ${long >= 0 ? 'E' : 'W'}`;
 }
+
+export function compareFirestoreKeys(fieldA: string, fieldB: string): number {
+  return fieldA.localeCompare(fieldB);
```

Comment:
Please add a TODO comment like `// TODO: Use UTF-8 encoded byte order instead to match Firestore production behavior` with a link to docs: https://firebase.google.com/docs/firestore/manage-data/data-types

Reply:
Done it